File: FizzBuzz/Approach1/FizzBuzz.cs
namespace FizzBuzzWithATwist.Approach1;

public static class FizzBuzz
{
    public static string[] FizzBuzzIt(int[] array)
    {
        if (array.Length >= 2 && array[0] > array[1])
        {
            return array.Select(r => r % 15 == 0 ?
                "zzub zzif" :
                r % 5 == 0 ?
                    "zzub" :
                    r % 3 == 0 ?
                        "zzif" :
                        r.ToString()
            ).ToArray();
        }
        else
        {
            return array.Select(r => r % 15 == 0 ?
                "fizz buzz" :
                r % 5 == 0 ?
                    "buzz" :
                    r % 3 == 0 ?
                        "fizz" :
                        r.ToString()
            ).ToArray();
        }
    }
}

File: FizzBuzzPy/FizzBuzz.py
def FizzBuzzIt(input: int):
    output: str = ''

    if input % 3 == 0:
        output += 'Fizz'

    if input % 5 == 0:
        output += 'Buzz'

    if not output:
        output = str(input)

    return output

def FizzBuzz100(number: int):
    if number == 1:
        return [FizzBuzzIt(number)]
    if number == 2:
        return [FizzBuzzIt(1), FizzBuzzIt(2)]
    return []